glommio/src/multitask.rs

```diff
@@ -31,7 +31,7 @@ use std::task::{Context, Poll};
 /// Once a `Runnable` is run, it "vanishes" and only reappears when its future is woken. When it's
 /// woken up, its schedule function is called, which means the `Runnable` gets pushed into a task
 /// queue in an executor.
-pub type Runnable = task_impl::Task<()>;
+pub type Runnable = task_impl::Task;
```

Comment:
No need to change anything because this is not related. But as I read this, I wonder if we shouldn't just get rid of Runnable now ? Or at least make it non-public.

It's just a task.

Reply:
I wondered the same. After some reflection, I came to the opinion that Runnable as for now is part of the public API and  task_impl::Task is more like the internal type. So I personally still like this type of aliasing.